contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -48,8 +59,7 @@ ConfigManager::ConfigManager(
 void ConfigManager::Init(RolloutApplyFunction rollout_apply_function) {
   rollout_apply_function_ = rollout_apply_function;
 
-  if (global_context_->service_name().empty() ||
-      global_context_->config_id().empty()) {
+  if (global_context_->service_name().empty()) {
```

Comment:
this case should be handled by start_esp.py.   We don't need the whole Init() call.  I suggested we remove it.   Or the only purpose of Init() is to start a periodic timer to call OnTimer(). if strategy=MANAGED.